ros2_foxglove_bridge/src/ros2_foxglove_bridge.cpp

```diff
@@ -444,9 +444,11 @@ class FoxgloveBridge : public rclcpp::Node {
     }
 
     try {
+      const bool isReliableSubscription = qos.reliability() == rclcpp::ReliabilityPolicy::Reliable;
```

Comment:
This does not work well when testing this with a realsense bag file where all topics (even point clouds & raw image topics) are published with `ReliabilityPolicy::Reliable`

I believe we need indeed a parameter with which the user can control which topics must always be send

Reply:
Separately from deciding whether we need a parameter for topic reliability overrides, it sounds like there is a bug somewhere in the pipeline of recording MCAP data and playing it back in ROS 2 with the same QoS policy it was originally published with. Can you file a bug ticket with your repro steps?

Reply:
I did not record the mcap file myself, so I can't provide reproduction steps. The Mcap recording seems to be fine though. I believe that the realsense node advertised topics as reliable (not sure if that is the default)

For reference, below the output of `mcap list channels`: 

```
/rosout                                      reliability: 1  durability: 1
/tf_static                                   reliability: 1  durability: 1
/camera/aligned_depth_to_color/camera_info   reliability: 1  durability: 2
/camera/aligned_depth_to_color/color/points  reliability: 1  durability: 2
/camera/aligned_depth_to_color/image_raw     reliability: 1  durability: 2
/camera/depth/image_rect_raw                 reliability: 1  durability: 2
/camera/color/camera_info                    reliability: 1  durability: 2
/camera/depth/camera_info                    reliability: 1  durability: 2
/camera/color/image_raw                      reliability: 1  durability: 2
```

`reliability: 1` maps to [rclcpp::ReliabilityPolicy::Reliable](https://docs.ros2.org/galactic/api/rclcpp/namespacerclcpp.html#a4f037ff6e64986ad3e4ee883e0b2ed2d) (by default, the value of the first enumerator is zero if it is implicitly defined)